src/server/api.php

```diff
@@ -924,7 +924,7 @@ function get_dengue_nowcast($locations, $epiweeks) {
 }
 
 // queries the `covidcast` table.
-//   $source (required): name of upstream data souce
+//   $endpoint (required): name of data endpoint; 'source' also accepted here for backwards-compatibility
```

Comment:
```suggestion
//   $source (required): name of upstream data source
```

Reverting 40f63f9; this was an unrelated typo fix